practice_9.py

```diff
@@ -0,0 +1,9 @@
+d=[]
```

Comment:
Display the exit condition to the user. For eg, print("press enter to stop inputting")